embark/requirements.txt

```diff
@@ -3,6 +3,12 @@ uWSGI
 ipython
 django-dotenv
 pycodestyle
+channels
+rx
+daphne
+inotify_simple
+inotify-simple
```

Comment:
Both are needed? What is the difference?

Reply:
im not sure. the documentation page says with '_' but in pycharm you only can install it with '-'. Is it a problem to keep them both?